package.json

```diff
@@ -6,14 +6,11 @@
   "private": true,
   "scripts": {
     "test": "yarn clean && yarn test:jest",
-    "test:jest":
-      "cross-env NODE_ENV=test nyc --reporter=html --reporter=lcov jest --config tests/setup/jest.config.js",
-    "test:coverage":
-      "nyc report --reporter=text-lcov > coverage.lcov && codecov --token=33eb1170-427a-4025-a7f3-9b2326f57bb0",
+    "test:jest": "cross-env NODE_ENV=test nyc --reporter=html --reporter=lcov jest --config tests/setup/jest.config.js",
```

Comment:
Just so you know, prettier will format this on a new line since the lines are longer than 80/120 characters. Is that a big issue? If so find the rule and disable it. It's ok with me but if it irritates you you probably want to disable it because when I save package json or use prettier it'll reformat it.

Reply:
That just happened when I saved - I didn't do it